tests/test_model.py

```diff
@@ -363,6 +368,546 @@ def test_model_export_trace_model(self):
             except SherlockModelServiceError as e:
                 pytest.fail(str(e))
 
+    def test_export_FEA_model(self):
+        channel_param = "127.0.0.1:9090"
+        channel = grpc.insecure_channel(channel_param)
+        model = Model(channel)
+
+        if platform.system() == "Windows":
+            temp_dir = os.environ.get("TEMP", "C:\\TEMP")
+        else:
+            temp_dir = os.environ.get("TEMP", "/tmp")
+        path = os.path.join(temp_dir, "export.wbjn")
+
+        if model._is_connection_up():
```

Comment:
There should be tests that don't require a connection for all of the code in the method that doesn't require a connection.